### PR TITLE
add keep_alive option

### PR DESCRIPTION
**Pull request by 0xRavenBlack**
- add keep_alive option (controls how long the model will stay loaded into memory following the request (default: 5m))
[Ollama API Description](https://github.com/ollama/ollama/blob/main/docs/api.md)

### CODE REVIEW

**Comment by stavsap** [comment]
can you please add description to https://github.com/stavsap/comfyui-ollama/blob/main/CHANGE_LOG.md?

**Comment by 0xRavenBlack** [comment]
@stavsap ok, its done :)

File: CompfyuiOllama.py [hunk 1]
```diff
@@ -61,7 +62,7 @@ def ollama_vision(self, images, query, debug, url, model):
 
 """)
 
-        response = client.generate(model=model, prompt=query, images=images_b64)
+        response = client.generate(model=model, prompt=query, images=images_b64, keep_alive=str(-keep_alive) + "m")
```
Comment (stavsap) [inline CompfyuiOllama.py:65]: why is str conversion is with negative value: -keep_alive ?

Comment (0xRavenBlack) [reply CompfyuiOllama.py]: @stavsap You're right that doesn't make sense, I read that somewhere on reddit I think.  (I fixed it)

**Pull request merged**